### PR TITLE
Add validator command

### PR DESCRIPTION
**Pull request by andocodes**

### CODE REVIEW

File: src/network/add_validators.go [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+		// TODO Creation of new keystore didn't work reestablish old one
+		return
+	}
+
+	// TODO delete old keystore
+}
+
+func (av *AddValidatorProcess) setupAddition() error {
+	credentials := av.configs.ValidatorCredentials
+	fmt.Println("You are currently have:")
```
Comment (NandoTheessen) [inline src/network/add_validators.go]: Small typo here :)

File: src/network/add_validators.go [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+	err := av.configs.Save()
+	if err != nil {
+		return err
+	}
+
+	err = av.configs.ValidatorCredentials.GenerateDepositDataWithRange(av.configs.DepositDetails, oldCredentials.ValidatorRange())
+	if err != nil {
+		return err
+	}
+	password := "asdasd"
```
Comment (NandoTheessen) [inline src/network/add_validators.go]: I imagine this need to be changed

File: src/network/add_validators.go [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+func (av *AddValidatorProcess) createNewKeystore() error {
+	oldCredentials := av.configs.ValidatorCredentials
+	oldCredentials.ValidatorIndexTo = av.newNumberOfValidators()
+	av.configs.ValidatorCredentials = oldCredentials
+	err := av.configs.Save()
+	if err != nil {
+		return err
+	}
+
+	err = av.configs.ValidatorCredentials.GenerateDepositDataWithRange(av.configs.DepositDetails, oldCredentials.ValidatorRange())
```
Comment (NandoTheessen) [inline src/network/add_validators.go:101]: Is the ValidatorRange on oldCredentials updated?

**Review by NandoTheessen — CHANGES_REQUESTED** [review]

File: src/network/add_validators.go (hunk 3 above)
Comment (andocodes) [reply src/network/add_validators.go:101]: Updated a few lines above! - `oldCredentials.ValidatorIndexTo = av.newNumberOfValidators()`

File: src/network/add_validators.go (hunk 2 above)
Comment (andocodes) [reply src/network/add_validators.go]: Good catch!

**Review by NandoTheessen — APPROVED** [review]

**Pull request merged**